src/client/index.js

```diff
@@ -34,4 +35,19 @@ module.exports.YotiClient = class YotiClient {
   performAmlCheck(amlProfile) {
     return amlService.performAmlCheck(amlProfile, this.pem, this.applicationId);
   }
+
+  /**
+   * @desc given a dynamic sharing request get a custom qr code denoted by the
+   * policy stated in the request.
+   * @param {*} dynamicSharingRequest is the request that contains the policy.
+   * @returns {Promise} constaining a dynamic
+   */
+  getDynamicPolicyQRCodeLink(dynamicPolicyRequest) {
+    // Build a dynbanic service, that gets the service
+    return dynamicPolicyService.getDynamicPolicy(
```

Comment:
same here --> `dynamicAttributeList`.
I'll not mention it in the remainder, but please update any reference to `policy` with `attributeList`